modules/gitops-disabling-the-creation-of-the-default-cluster-roles-for-the-cluster-scoped-instance.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-disabling-the-creation-of-the-default-cluster-roles-for-the-cluster-scoped-instance_{context}"]
+= Disabling the creation of the default cluster roles for the cluster-scoped instance
+
+To add or remove permissions to cluster-wide resources, as needed, you must disable the creation of the default cluster roles for the cluster-scoped instance by editing the `ArgoCD` custom resource (CR) YAML file.
+
+.Procedure
+
+. Set the value of the `.spec.defaultClusterScopedRoleDisabled` field to `true`:
++
+.Example Argo CD custom resource
+
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example <1>
+  namespace: spring-petclinic <2>
+# ...
+spec:
+  defaultClusterScopedRoleDisabled: true <3>
+# ...
+----
+<1> The name of the default Argo CD instance.
+<2> The namespace where you want to run the Argo CD instance.
+<3> The flag value that disables the creation of the default cluster roles for the cluster-scoped instance.
++
+.Sample output
+[source,terminal]
+----
+argocd.argoproj.io/example configured
+----
+
+. Verify that the {gitops-title} Operator has deleted the default cluster roles and cluster role bindings for the {gitops-shortname} control plane components by running the following commands:
++
+[source,terminal]
+----
+$ oc get ClusterRoles -l app.kubernetes.io/part-of=argocd
+----
++
+[source,terminal]
+----
+$ oc get ClusterRoleBindings -l app.kubernetes.io/part-of=argocd
```

Comment:
I think this could create confusion, if there is another cluster-scoped instance running, which is using default ClusterRoles then `oc get ClusterRoles -l app.kubernetes.io/part-of=argocd` command will list those as well, instead we can be specific that default ClusterRoles targeting user's instance should be deleted.
`oc get ClusterRoles/<argocd_name>-<argocd_namespace>-<control_plane_component>`